nym-vpn-apple/MixnetLibrary/Package.swift

```diff
@@ -23,8 +23,8 @@ let package = Package(
         ),
         .binaryTarget(
             name: "NymVpnLib",
-            url: "https://github.com/nymtech/nym-vpn-client/releases/download/nym-vpn-core-v0.1.16/nym-vpn-core-v0.1.16_ios_universal.zip",
-            checksum: "41a5ecb8b0ab7cc2e6130179863761d12bcc988eda6c269cb00bd01a43449741"
+            url: "https://github.com/nymtech/nym-vpn-client/releases/download/nym-vpn-core-nightly/nym-vpn-core-v0.1.17-dev_ios_universal.zip",
+            checksum: "66a830cd46b912df1c11704b487f7a661299ff4583d142770ae04c0ceeaaad5c"
```

Comment:
This will break after the next nightly build?

Reply:
This will break. I believe this is a temp fix to get the Testflight out the door.

Reply:
If you need a custom build of the vpn lib that is non-temporary you can use a named tag like `nym-vpn-core-ios-testflight`. But maybe if we merge this on Monday we can create a new release core then and use that?

Reply:
That would be ideal 👍

Reply:
we'll fix this with a follow up pr